t/utils/utils.go

```diff
@@ -95,10 +96,20 @@ func LoadFromFile(filename string) string {
 	return buf.String()
 }
 
+// LightEthereumProvider interface to be used on EnsureNodeSync
```

Comment:
This interface has a lot of unrelated methods. We can leave it as it is for now but we should add a `TODO` comment to get it fixed.

Reply:
done